release notes/v0.47.0.md

```diff
@@ -0,0 +1,113 @@
+
+### Higher precision for Trend metrics in Cloud [#3302](https://github.com/grafana/k6/pull/3302)
+
+k6 Cloud is now able to store and visualize Trend metrics up to 3 digits of precision for decimal numbers.
+
+### Docker images for ARM64 architecture [#3320](https://github.com/grafana/k6/pull/3320)
+
+k6 now builds and pushes dedicated Docker images for ARM64. Check k6's [tags page](https://hub.docker.com/r/grafana/k6/tags) on DockerHub for details. 
+
+### Improved the browser module's cookie API
+
+The browser module now provides a more complete and robust API for handling cookies. We stabilized the cookie API by defining a new [`Cookie` class](https://k6.io/docs/javascript-api/k6-experimental/browser/browsercontext/cookie) ([browser#1008](https://github.com/grafana/xk6-browser/pull/1008), [browser#1030](https://github.com/grafana/xk6-browser/pull/1030)) that can be used while creating and retrieving cookies.
+
+We added a new [`browserContext.cookies([urls])`](https://k6.io/docs/javascript-api/k6-experimental/browser/browsercontext/cookies/) ([browser#1005](https://github.com/grafana/xk6-browser/pull/1005)) method that returns all cookies from the current [browser context](https://k6.io/docs/javascript-api/k6-experimental/browser/browsercontext). The new API also supports filtering cookies by URL ([browser#1016](https://github.com/grafana/xk6-browser/pull/1016)). We also fixed a bug where the `expires` field was not being set correctly while adding cookies using the [`context.addCookie()`](https://k6.io/docs/javascript-api/k6-experimental/browser/browsercontext/addcookies/) method. Lastly, we fixed the existing [`context.clearCookies()`](https://k6.io/docs/javascript-api/k6-experimental/browser/browsercontext/clearcookies) method to clear all cookies from the current [browser context](https://k6.io/docs/javascript-api/k6-experimental/browser/browsercontext).
+
+```js
+const context = browser.newContext();
+context.addCookies([
+    {name: 'foo', value: 'bar', url: 'https://test.k6.io'},
+    {name: 'baz', value: 'qux', url: 'https://grafana.com'},
+]);
+
+const cookies = context.cookies('https://test.k6.io');
+console.log(cookies.length);         // 1
+console.log(cookies[0].name);        // foo
+console.log(cookies[0].value);       // bar
+
+context.clearCookies();
+console.log(context.cookies.length); // 0
+```
+
+### Add support for browser module's `page.on('console')` [browser#1006](https://github.com/grafana/xk6-browser/pull/1006)
+
+Allows users to register a handler to be executed every time the `console` API methods are called from within the page's JavaScript context. The arguments passed into the handler are defined by the [ConsoleMessage](https://k6.io/docs/javascript-api/k6-experimental/browser/consolemessage/) class.
```

Comment:
```suggestion
This support allows users to register a handler to be executed every time the `console` API methods are called from within the page's JavaScript context. The arguments passed into the handler are defined by the [ConsoleMessage](https://k6.io/docs/javascript-api/k6-experimental/browser/consolemessage/) class.
```

Reply:
I think the original message is good enough, thanks.